src/config.c

```diff
@@ -3240,8 +3240,8 @@ standardConfig static_configs[] = {
     createLongLongConfig("busy-reply-threshold", "lua-time-limit", MODIFIABLE_CONFIG, 0, LONG_MAX, server.busy_reply_threshold, 5000, INTEGER_CONFIG, NULL, NULL),/* milliseconds */
     createLongLongConfig("cluster-node-timeout", NULL, MODIFIABLE_CONFIG, 0, LLONG_MAX, server.cluster_node_timeout, 15000, INTEGER_CONFIG, NULL, NULL),
     createLongLongConfig("cluster-ping-interval", NULL, MODIFIABLE_CONFIG | HIDDEN_CONFIG, 0, LLONG_MAX, server.cluster_ping_interval, 0, INTEGER_CONFIG, NULL, NULL),
-    createLongLongConfig("slot-migration-pause-write-max-gap-size", NULL, MODIFIABLE_CONFIG, 0, LLONG_MAX, server.asm_pause_write_max_gap_size, 1*1024*1024, MEMORY_CONFIG, NULL, NULL), /* 1MB */
-    createLongLongConfig("slot-migration-pause-write-timeout", NULL, MODIFIABLE_CONFIG, 0, LLONG_MAX, server.asm_pause_write_timeout, 10*1000, INTEGER_CONFIG, NULL, NULL), /* 10 seconds */
+    createLongLongConfig("slot-migration-handoff-max-lag-size", NULL, MODIFIABLE_CONFIG, 0, LLONG_MAX, server.asm_handoff_max_lag_size, 1*1024*1024, MEMORY_CONFIG, NULL, NULL), /* 1MB */
```

Comment:
maybe `slot-migration-handoff-max-lag-bytes` is slightly better alternative?

Reply:
sounds good, AI also suggest to use this one.

changed